src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -324,8 +324,8 @@ func (c *Client) DeleteShelf(ctx context.Context, req *librarypb.DeleteShelfRequ
 }
 
 // MergeShelves merges two shelves by adding all books from the shelf named
-// `other_shelf_name` to shelf `name`, and deletes
-// `other_shelf_name`. Returns the updated shelf.
+// other_shelf_name to shelf name, and deletes
+// other_shelf_name. Returns the updated shelf.
```

Comment:
I feel like stripping out the backticks and not replacing them with anything produces potential ambiguities in the text.